libraries/lib-time-and-pitch/StaffPadTimeAndPitch.cpp

```diff
@@ -82,9 +79,9 @@ void StaffPadTimeAndPitch::GetSamples(float* const* output, size_t outputLen)
          const auto numSamplesToGet =
             std::min({ maxBlockSize, numOutputSamplesAvailable,
                        static_cast<int>(outputLen - numOutputSamples) });
-         const auto buffer =
-            GetOffsetBuffer(output, mNumChannels, numOutputSamples);
-         mTimeAndPitch->retrieveAudio(buffer.data(), numSamplesToGet);
+         float *buffer[2]{};
```

Comment:
Some sanity checks that mNumChannels <= 2 won't hurt, `assert` will do

Reply:
Done